go/vt/vtctl/workflow/server_test.go

```diff
@@ -2887,3 +2888,19 @@ func TestMaterializeAddTables(t *testing.T) {
 		})
 	}
 }
+
+func TestMoveTablesCreateSourceEqualsTarget(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+	defer cancel()
+
+	ts := memorytopo.NewServer(ctx, "cell1")
+
+	_, err := NewServer(vtenv.NewTestEnv(), ts, nil).MoveTablesCreate(ctx, &vtctldatapb.MoveTablesCreateRequest{
+		SourceKeyspace: "ks1",
+		TargetKeyspace: "ks1",
+		Workflow:       "wf1",
+	})
+
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "source and target keyspace must be different")
```

Comment:
You can 2 lines together with `require.ErrorContains`. That checks for it both being an error and having that be part of the message.